documentation/chatmeleon-swagger.yaml

```diff
@@ -22,22 +22,14 @@ servers:
 tags:
   - name: conversations
   - name: messages
-  - name: users
 paths:
-  /users/{userId}/conversations:
+  /users/conversations:
```

Comment:
Should this be `/users/conversations` or just `conversations`?

Reply:
ChatGPT told me that getting all conversations of a user would be more related to /users than /conversations. So it suggested using users/conversations indicating that this route has something with the users more. But im flexible on this one so either way is fine.

Reply:
It would apply if it were `/users/{userId}/conversations` I guess. We changed the route to only get conversations of the current user

Reply:
yea makes sense so i think we gonna go for `/conversations`